bufferserver/src/main/java/com/datatorrent/bufferserver/server/Server.java

```diff
@@ -73,9 +73,12 @@
   private InetSocketAddress address;
   private final ExecutorService serverHelperExecutor;
   private final ExecutorService storageHelperExecutor;
+  private boolean backPressureEnabled;
 
   private byte[] authToken;
 
+  private static final boolean BACK_PRESSURE_ENABLED = !Boolean.getBoolean("BUFFERSERVER.BACK_PRESSURE_DISABLE");
```

Comment:
The assumption was that the backpressure will be always enabled and will be managed at the individual stream level after the second PR. It is not necessary (and not desirable) to have programmatic configurability at the Server level. Should such functionality be introduced, it will be harder to drop it later.